src/fontra/views/editor/editor.html

```diff
@@ -316,13 +316,17 @@
 
     .sidebar-resize-gutter {
       height: 100%;
-      width: 8px;
+      width: 6px;
       cursor: var(--resize-cursor);
       position: absolute;
     }
 
     .sidebar-container.left .sidebar-resize-gutter {
-      right: 0;
+      right: -4px;
```

Comment:
This should be half of the width. Same for right.

Did you try with a width of 4px? Did you find it too narrow?

Reply:
Ok. I made it 4px with, -2px for offsets. Looks good to me.